Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml

```diff
@@ -202,7 +205,49 @@ tasks:
     ignoreworker: false
     skipunavailable: true
     quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: a9dd368b-4676-40e3-85a6-e027c062b6c6
+    type: playbook
+    task:
+      id: a9dd368b-4676-40e3-85a6-e027c062b6c6
+      version: -1
+      name: Cortex XDR - Retrieve File by sha256
+      playbookName: Cortex XDR - Retrieve File by sha256
```

Comment:
this playbook won't work properly - as it was designed for a dedicated use case. The retrieve process requires an Endpoint ID and a dedicated file path which is passed through an alert. This means that you can't retrieve a sample only by hash ( which is the main intent of the parent PB).